community-operators/triggermesh/triggermesh.v0.0.1.clusterserviceversion.yaml

```diff
@@ -0,0 +1,167 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  annotations:
```

Comment:
nit: Please add additional annotations as described in this doc: https://github.com/operator-framework/community-operators/blob/master/docs/required-fields.md

Reply:
added

Reply:
Hi, you still missing here required fields as `categories`, `support`, `description`

Reply:
Just added, thanks